pkg/controller/plan/migration.go

```diff
@@ -1143,129 +1147,122 @@ func (r *Migration) setRunning(vm *plan.VMStatus, running bool) (err error) {
 	return
 }
 
-//
-// Determine whether any of the DataVolumes haven't been bound yet
-// because they are waiting for first consumer.
-func (r *Migration) waitingForFirstConsumer(vm *plan.VMStatus) (waiting bool, err error) {
-	if r.vmMap == nil {
-		r.vmMap, err = r.kubevirt.VirtualMachineMap()
-		if err != nil {
-			return
-		}
-	}
-	var vmCr VirtualMachine
-	found := false
-	if vmCr, found = r.vmMap[vm.ID]; !found {
-		msg := "VirtualMachine CR not found."
-		vm.AddError(msg)
-		return
-	}
-	for _, dv := range vmCr.DataVolumes {
-		if dv.Status.Phase == cdi.WaitForFirstConsumer {
-			waiting = true
-			return
-		}
-	}
-
-	return
-}
-
 //
 // Update the progress of the appropriate disk copy step. (DiskTransfer, Cutover)
 func (r *Migration) updateCopyProgress(vm *plan.VMStatus, step *plan.Step) (err error) {
-	if r.vmMap == nil {
-		r.vmMap, err = r.kubevirt.VirtualMachineMap()
-		if err != nil {
-			return
-		}
-	}
-
-	var vmCr VirtualMachine
-	found := false
-	if vmCr, found = r.vmMap[vm.ID]; !found {
-		msg := "VirtualMachine CR not found."
-		vm.AddError(msg)
-		return
-	}
-
 	var pendingReason string
 	var pending int
 	var completed int
```

Comment:
not related to this PR, but maybe we should rename this variable? it has the same name as the one in L1221, but these are different variables

Reply:
yeah, it can confuse. here it's the tasks (number of disks) and withing the for loop it's the amount of data transferred.